### ISSUE TITLE
AccessCompiler Not Using Expected Comparison Operator

### ISSUE THREAD

**Issue by the-wondersmith** [gordthompson/sqlalchemy-access#6]
I'm not sure if this is unique to my situation or not, but sqlalchemy-access appears to not be substituting the correct comparison operator when compiling not-equals statements.

It leaves the default ` != ` operator in place (instead of using the apparently expected ` <> `), which causes pyodbc to throw a ProgrammingError in response to the MS Access ODBC driver throwing a syntax error:

> [Microsoft][ODBC Microsoft Access Driver] Syntax error (missing operator) in query expression

This can apparently be prevented / remedied by applying the following code before initializing the SQLAlchemy DB connection:

```python
from sqlalchemy.sql.compiler import OPERATORS
 from sqlalchemy.sql.operators import ne  

OPERATORS[ne] =  " <> "
```

I'd submit a pull request to address it, but I'm not 100% it's not a local-only issue, and I have no way to test it. Please let me know if I should do so or not.

**Comment by gordthompson** [comment]
It's not just you. If you could submit a PR that would be great!

**Comment by the-wondersmith** [comment]
> It's not just you. If you could submit a PR that would be great!

[PR submitted](https://github.com/sqlalchemy/sqlalchemy-access/pull/7)

**Comment by gordthompson** [comment]
Fixed in #7
